src/components/Tabs/tabs.css

```diff
@@ -32,20 +32,11 @@
     }
     .lil-tabs > .lil-tab {
         min-width: 150px;
-        @apply lil-text-base lil-mr-2 lil-rounded-md;
-    }
-    .lil-tabs .lil-tab-active {
-        @apply lil-bg-blue-350;
-    }
-    .lil-tab-active .lil-tab-heading-text {
-        @apply lil-text-white;
```

Comment:
is it not causing regression ?

Reply:
Only changed active styling for small screen here, checked for both home & individuals